helpers/installcredprovider.ps1

```diff
@@ -80,7 +80,7 @@ if (!$releaseId) {
 $releaseUrl = [System.IO.Path]::Combine($releaseUrlBase, $releaseId)
 $releaseUrl = $releaseUrl.Replace("\","/")
 
-$zipFile = "Microsoft.NetCore2.NuGet.CredentialProvider.zip"
+$zipFile = "Microsoft.NetCore3.NuGet.CredentialProvider.zip"
```

Comment:
This appears to have broken use of this script.
> Unable to resolve the Credential Provider zip file from https://api.github.com/repos/Microsoft/artifacts-credprovider/releases/latest

It turns out that this script looks for a file by this name in the latest release from this repo, but [the latest release](https://github.com/microsoft/artifacts-credprovider/releases/tag/v0.1.28) only has a file by the name of Microsoft.NetCore**2**.NuGet.CredentialProvider.zip.

This is breaking by cloud builds. Please either issue a new release with a Microsoft.NetCore3 file, or revert this change.

Reply:
Thank you @AArnott for calling it out. We reverted the change until we are ready to release the new version.

Reply:
Confirmed. The script is working now. Thank you.